cocos/3d/CCMesh.cpp

```diff
@@ -350,6 +350,7 @@ void Mesh::draw(Renderer* renderer, float globalZOrder, const Mat4& transform, u
 
 
     _meshCommand.setSkipBatching(isTransparent);
+    _meshCommand.setTransparent(isTransparent);
```

Comment:
yes, this line is needed. It decides which render queue this command be placed in.